tendrl/gluster_integration/message/process_events.py

```diff
@@ -13,7 +13,7 @@ def process_events():
                 # this perticular event is recovery event
                 # so process this event and delete it
                 event_utils.emit_event(
-                    event.context.split("_")[0],
+                    event.context.split("-")[0],
```

Comment:
Is it good to use hyphon as separator? What are changes are having hyphon in data parts itself. I feel rather we should pipe may be

Reply:
ack. Good suggestion. Using `|` now